src/server.c

```diff
@@ -2911,46 +2910,75 @@ void resetErrorTableStats(void) {
 
 /* ========================== Redis OP Array API ============================ */
 
-void redisOpArrayInit(redisOpArray *oa) {
-    oa->ops = NULL;
-    oa->numops = 0;
-    oa->capacity = 0;
-}
-
 int redisOpArrayAppend(redisOpArray *oa, int dbid, robj **argv, int argc, int target) {
```

Comment:
maybe it should just return the index of the appended op, instead of the total number of ops?